docs/eventing/getting-started.md

```diff
@@ -0,0 +1,524 @@
+
+    ---
+
+    # Service pointing at the previous Deployment. This will be the target for event
+    # consumption.
+    kind: Service
+    apiVersion: v1
+    metadata:
+      name: hello-display
+    spec:
+      selector:
+        app: hello-display
+      ports:
+      - protocol: TCP
+        port: 80
+        targetPort: 8080
+    END
+    ```
+
+2. To deploy the `goodbye-display` consumer to your cluster, run the following command:
+
+    ```sh
+    kubectl --namespace event-example apply --filename - << END
+    apiVersion: apps/v1
+    kind: Deployment
+    metadata:
+      name: goodbye-display
+    spec:
+      replicas: 1
+      selector:
+        matchLabels: &labels
+          app: goodbye-display
+      template:
+        metadata:
+          labels: *labels
+        spec:
+          containers:
+            - name: event-display
+              # Source code: https://github.com/knative/eventing-contrib/blob/release-0.6/cmd/event_display/main.go
+              image: gcr.io/knative-releases/github.com/knative/eventing-sources/cmd/event_display@sha256:37ace92b63fc516ad4c8331b6b3b2d84e4ab2d8ba898e387c0b6f68f0e3081c4
+
+    ---
+
+    # Service pointing at the previous Deployment. This will be the target for event
+    # consumption.
+    kind: Service
+    apiVersion: v1
+    metadata:
+      name: goodbye-display
+    spec:
+      selector:
+        app: goodbye-display
+      ports:
+      - protocol: TCP
+        port: 80
+        targetPort: 8080
+    END
+    ```
+
+3. Just like you did with the `Broker`, verify that your event consumers are working by running the following command:
+
+    ```sh
+    kubectl --namespace event-example get deployments hello-display goodbye-display
+    ```
+
+    This lists the `hello-display` and `goodbye-display` consumers that you deployed:
+
+    ```sh
+    NAME           DESIRED   CURRENT   UP-TO-DATE   AVAILABLE   AGE
+    hello-display    1         1         1            1           26s
+    NAME           DESIRED   CURRENT   UP-TO-DATE   AVAILABLE   AGE
+    goodbye-display    1         1         1            1           16s
+   ```
+
+    The number of replicas in your **DESIRED** column should match the number of replicas in your **AVAILABLE** column, which might take a few minutes. If after two minutes the numbers do not match, then see the [Debugging Guide](./debugging/README.md) to help troubleshoot the issue.
+
+### Creating `Triggers`
+
+A [Trigger](./broker-trigger.md#trigger) defines the events that you want each of your event consumers 
+to receive. Your `Broker` uses triggers to forward events to the right consumers. Because all events 
+in Knative align with the [CloudEvents](https://cloudevents.io/) specification, each event in Knative must include 
+the `type` and `source` attributes. Each trigger uses the `type` and `source` attributes to match events 
```

Comment:
This is no longer true (even though it was when you started to write this document :))
Change the text to:
"All events in Knative align with the [CloudEvents](https://cloudevents.io/) specification. Each Trigger can specify a filter to select relevant events based on the [Cloud Event context attributes](https://github.com/cloudevents/spec/blob/master/spec.md#context-attributes).